notebooks/a00_introduction_to_geomstats.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
<p>All figures should be referenced in the text. The figures (a), (b) above are not and should be.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>fixed</p>